tests/v2/validation/provisioning/provisioning_rke1_driver_test.go

```diff
@@ -0,0 +1,185 @@
+package provisioning
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/rke1/nodetemplates"
+	aws "github.com/rancher/rancher/tests/framework/extensions/rke1/nodetemplates/aws"
+	azure "github.com/rancher/rancher/tests/framework/extensions/rke1/nodetemplates/azure"
+	linode "github.com/rancher/rancher/tests/framework/extensions/rke1/nodetemplates/linode"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/rancher/rancher/tests/framework/pkg/wait"
+	"github.com/rancher/rancher/tests/integration/pkg/defaults"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type RKE1NodeDriverProvisioningTestSuite struct {
+	suite.Suite
+	client             *rancher.Client
+	session            *session.Session
+	kubernetesVersions []string
+	cnis               []string
+	providers          []string
+}
+
+func (r *RKE1NodeDriverProvisioningTestSuite) TearDownSuite() {
+	r.session.Cleanup()
+}
+
+func (r *RKE1NodeDriverProvisioningTestSuite) SetupSuite() {
+	testSession := session.NewSession(r.T())
+	r.session = testSession
+
+	clustersConfig := new(Config)
+	config.LoadConfig(ConfigurationFileKey, clustersConfig)
+
+	r.kubernetesVersions = clustersConfig.KubernetesVersions
+	r.cnis = clustersConfig.CNIs
+	r.providers = clustersConfig.Providers
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(r.T(), err)
+
+	r.client = client
+
+	enabled := true
+	var testuser = AppendRandomString("testuser-")
+	user := &management.User{
+		Username: testuser,
+		Password: "rancherrancher123!",
```

Comment:
I'm sure we do this in other tests, but I feel that we shouldn't expose our test passwords at all. Can we hide these, or just leave them for the user to specify (this would be a new ticket).

Reply:
I think it would be better to have the user specify. This is present in our other tests as well, so once we figure out best way to do that, I think we could make the needed adjustments across the board.